### ISSUE TITLE
Thrallmar reputation in Blood Furnace

### ISSUE THREAD

**Issue by Hyp3rViper** [dalaranwow/dalaran-wow#5223]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The trash mobs in the Hellfire Citadel: Blood Furnace instance give only 1 Thrallmar reputation
**Expected behaviour**: Tell us what should happen instead
They should be giving 5 reputation per kill, as evidenced by a Wowhead comment from patch 2.4.0 by Nazaref
**Steps to reproduce the problem**:

1. Go to Blood Furnace
2. Kill a trash mob
3. Get only 1 reputation for Thrallmar per mob

**Include proofs for this behaviour**

https://imgur.com/a/NlVYg

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and thank you for the report @Hyp3rViper.
Issue confirmed on PTR.
I would like to add that alliance characters also gain 1 reputation point towards honor hold (should be 5) with all creatures inside Blood Furnace. Exeptions are bosses and imps (http://www.wowhead.com/npc=19016/hellfire-familiar, http://www.wowhead.com/npc=17477/hellfire-imp).
Imps should continue to reward 1 reputation per kill and bosses 50 on normal.

Adding suggested fix for all creatures inside Blood Furnace without imps and bosses (they already reward the correct amount).

Note: @Hyp3rViper for future reports please fill the entire template so we can get all the needed information. In this report a list of all the creatures that should reward more reputation would of been great to have.

**Comment by TeridanDW** [comment]
Works as intended on PTR.
